src/components/maincontroller.ts

```diff
@@ -566,7 +578,7 @@ export function getMaxBitrate() {
 
         console.log('detecting bitrate');
 
-        detectBitrate($scope).then(
+        detectBitrate().then(
             function (bitrate) {
```

Comment:
Another thing that could be an arrow function

Reply:
yey :)